assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -68,6 +70,8 @@ type PropTypes = {|
   onThirdPartyPaymentDone: Token => void,
   checkoutFormHasBeenSubmitted: boolean,
   setCheckoutFormHasBeenSubmitted: () => void,
+  openDirectDebitPopUp: void => void,
```

Comment:
The usual pattern we use is `() => void`. Is there a reason for using `void => void`?

Reply:
no! thanks for pointing out, it should be `() => void`